lesson-12/main.js

```diff
@@ -0,0 +1,60 @@
+function dateArr() {
+  const remoteArray = fetch (
+    "https://fe-student-api.herokuapp.com/api/hotels/popular"
+  );
+  const jsonArr = remoteArray.then((itemArr) => itemArr.json());
+  jsonArr.then((arr) => htmlItem(arr));
+}
+dateArr();
+
```

Comment:
is it necessary to add files from lesson-12? 67 files changed - it is hard to check